### ISSUE TITLE
Review: Case studies in cloud migration: Netflix, Pinterest, and Symantec

### ISSUE THREAD

**Issue by jpronans** [Sairam1902/Cloud-Migration-Strategies-for-Enterprises#8]
https://increment.com/cloud/case-studies-in-cloud-migration/

**Comment by Sairam1902** [comment]
completed.

**Comment by jpronans** [comment]
So.. to close this one you either need to have decided 'yes' you are going to use it and include a citation like this
'''
@techreport{harrisPCAPCaptureFile2020,
  author = {Guy Harris and Michael Richardson},
  title = {PCAP Capture File Format},
  howpublished = {Working Draft},
  type = {Internet-Draft},
  number = {draft-gharris-opsawg-pcap-01},
  year = {2020},
  month = {December},
  institution = {IETF Secretariat},
}
  
'''

Or else a reason as to why you haven't used the reference.

**Comment by Sairam1902** [comment]
This reference style was used.

**Comment by jpronans** [comment]
What did you think of the paper, was it useful, yes or no?

**Comment by Sairam1902** [comment]
Yes, the paper was useful. It provided valuable insights into Netflix's cloud migration. It also highlighted the challenges, strategic decisions, and innovations. Therefore, it is helps to provide practical lessons for businesses considering cloud adoption.

**Comment by jpronans** [comment]
Fair enough, closing
